inst/tutorials/practical3-2/practical3-2.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Practical 3-2: Stochastic population growth model"
+title: "3-2: Stochastic population growth model"
```

Comment:
You seem to have reverted some of my changes? Was there a reason for that. All of the previous practicals have the word practical first...

Reply:
Sorry! I did have a merge conflict. But wasn’t given the option to resolve it. So I cloned the repo and added these files back in. Sorry! Feel free to change it back..

Reply:
Will do. No problem - I assumed something like that had happened.